SillySCP/Commands/StartVolunteer.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CommandSystem;
+using Exiled.API.Enums;
+using Exiled.API.Extensions;
+using Exiled.API.Features;
+using MEC;
+using PlayerRoles;
+using PlayerRoles.RoleAssign;
+using SillySCP.API.Features;
+
+namespace SillySCP.Commands
+{
+    [CommandHandler(typeof(RemoteAdminCommandHandler))]
+    public class StartVolunteer : ICommand
+    {
+        public string Command { get; } = "startvolunteer";
+        public string Description { get; } = "Start a volunteer prompt";
+        public string[] Aliases { get; } = new[] { "sv", "svol" };
+
+        private void CreateVolunteer(RoleTypeId oldRole)
+        {
+            Volunteers volunteer = new ()
+            {
+                Replacement = oldRole,
+                Players = new()
+            };
+            VolunteerSystem.Volunteers ??= new();
+            VolunteerSystem.Volunteers.Add(volunteer);
+            Map.Broadcast(10,
```

Comment:
If the volunteer is for a zombie, do a loop through all players and only broadcast to dead people.